### ISSUE TITLE
"Invalid LNDhub-URI" when try to using public domain

### ISSUE THREAD

**Issue by LOCHER-21** [BlueWallet/BlueWallet#7362]
iOS 18.1.1 (22B91) iPhone 15 Pro
BlueWallet ver 7.0.5 (build 1732553778 master)

Got the still the error of when I try to add my previous working LNDhub and the bug ist not fixed. The release notes say my last issue 7087/7090 related to this problem is fixed but it doesn't. If I use the local IP at my home network it works fine but if I want to use my public domain http://xxx.link:3000 just outputs "Invalid LNDhub-URI".

![Bildschirmfoto 2024-11-27 um 07 00 35 jpeg](https://github.com/user-attachments/assets/ff44f14d-7e58-46e8-a84a-ed912bb95001)

**Comment by Overtorment** [comment]
can you actually open your link from outer internet on the specified port?
i noticed `http` protocol, could it be that you actually ment `https` ?

**Comment by LOCHER-21** [comment]
Yes I can reach my LNDhub page on the public internet with http://xxx.link:3000. But when I scan the QR-Code on the website I get the error message. Same when I type it manually. This hole setup worked perfect till a BlueWallet update 3 or 4 months ago. I still have lightning wallet in my BlueWallet that no longer working cause it look like the format of my public domain is no longer valid. Very frustrating!

My LNDhub is only reachable on http and not on the https.

**Comment by marcosrdz** [comment]
Apple nor Google allow http connections unless pre-specified in the code.

**Comment by LOCHER-21** [comment]
Like I said. I had this system setup and running a for along time. The update to BlueWallet 7.0.0 broke the functionality on my setup.

**Comment by LOCHER-21** [comment]
Have now also tried https://xxx.link:3001/ but it doesn't work either. Same error message.
